 .hound.yml

```diff
@@ -1,3 +1,9 @@
+jshint:
```

Comment:
Remove this file cause is useless now.

Reply:
@justmesam, I have implemented the changes. Thank you.